README.md

```diff
@@ -1362,6 +1363,66 @@ require'nvim_lsp'.leanls.setup{}
     settings = {}
 ```
 
+## metals
+
+https://scalameta.org/metals/
+
+Scala language server with rich IDE features.
+`metals` can be installed via `:LspInstall metals`.
+
+Can be installed in Nvim with `:LspInstall metals`
+This server accepts configuration via the `settings` key.
+<details><summary>Available settings:</summary>
+
+- **`metals.customRepositories`**: `array`
+
+  Array items: `{type = "string"}`
+
+- **`metals.gradleScript`**: `string`
+
+  
+
+- **`metals.javaHome`**: `string`
+
+  
+
+- **`metals.mavenScript`**: `string`
+
+  
+
+- **`metals.millScript`**: `string`
+
+  
+
+- **`metals.sbtScript`**: `string`
+
+  
+
+- **`metals.scalafmtConfigPath`**: `string`
+
+  
+
+- **`metals.serverProperties`**: `array`
+
+  Array items: `{type = "string"}`
+
+- **`metals.serverVersion`**: `string`
+
+  Default: `"0.7.6"`
+
+</details>
+
+```lua
+require'nvim_lsp'.metals.setup{}
+
+  Default Values:
+    cmd = { "metals" }
+    filetype = { "scala", "sbt" }
```

Comment:
One last though. You probably don't need `sbt` in here. I'm assuming the fast majority of people will purposefully have `*.sbt` files marked as `scala`, which is recommended to do.

Reply:
I see. Thanks.

Reply:
I have updated it in metals.lua. The bot will be update the README automatically.